core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -1013,6 +1047,10 @@ public SELF withCreateContainerCmdModifier(Consumer<CreateContainerCmd> modifier
         return self();
     }
 
+    public void apply() {
```

Comment:
what does this method do?

Reply:
the method is called by the watcher, which is created with a GenericContainer. In BrowserWebDriverContainer, the implementation start recording in a file

Reply:
I've done some tests. 
- Using @ClassRule for BrowserWebDriverContainer is much faster just @Rule.
- Killing old binaries is faster than letting several recording on the container.

I've begun comparing your solution with this. On the tests I've written, the times are similar but I have only 2 tests at the moment, perhaps a little faster with my code. But I see a difference : 
- Times are almost the same on the 2 tests with your solution
- It seems to have a better time on the second time in my case
so, on many tests, a greater difference could appear